spec/requests/catalog_request_spec.rb

```diff
@@ -108,6 +108,15 @@
     end
   end
 
+  describe "collection-level bioghist notes" do
+    xit "renders the bioghist notes on the collection show page with <hr />" do
```

Comment:
This should not be failing (I am not certain what is triggering this, but should the last CircleCI build pass, I will remove this).